docs/step-by-step/step-06.md

```diff
@@ -0,0 +1,113 @@
+# Step 6. Collect metrics from more services and apps
+
+When Netdata _starts_, it auto-detects dozens of **data sources**, such as database servers, web servers, and more.
+
+To auto-detect and collect metrics from a source you just installed, you need to [restart
+Netdata](../getting-started.md#start-stop-and-restart-netdata).
+
+However, auto-detection only works if you installed the source using its standard installation
+procedure. If Netdata isn't collecting metrics after a restart, your source probably isn't configured
+correctly.
+
+Check out the [available data collection modulues](../Add-more-charts-to-netdata.md#available-data-collection-modules)
+to find the module for the source you want to monitor.
+
+## What you'll learn in this step
+
+We'll begin with an overview on Netdata's plugin architecture, and then dive into the following:
+
+-   [Netdata's plugin architecture](#netdatas-plugin-architecture)
+-   [Enable and disable plugins](#enable-and-disable-plugins)
+-   [Enable the Nginx module as an example](#example-enable-the-nginx-module)
+
+## Netdata's plugin architecture
+
+Many Netdata users never have to configure plugins or worry about which plugin orchestrator they want to use.
+
+But, if you want to configure plugins or write a collector module for your custom source, it's important to understand
+the underlying plugin architecture.
+
+By default, Netdata collects a lot of metrics every second using a lot of plugins. **Internal** plugins collect system
+metrics, **external** plugins collect non-system metrics, and **orchestrator** plugins support data collection modules.
+
+These modules are primarily written in [Go](../../collectors/go.d.plugin/) (`go.d`) and
+[Python](../../collectors/python.d.plugin/), although some use [Bash](../../collectors/charts.d.plugin/) (`charts.d`) or
+[Node.js](../../collectors/node.d.plugin/) (`node.d`).
+
+## Enable and disable plugins
+
+You don't need to explicitly enable plugins to auto-detect properly configured sources, but it's useful to know how to
+enable or disable them.
+
+One reason you might want to _disable_ plugins is to improve Netdata's performance on low-resource systems, like
+ephemeral nodes or edge devices.
```

Comment:
Maybe mention that disabling orchestrator plugins in particular can save significant amounts of resources?

Reply:
Done.